webpack/react_app/redux/actions/jobInvocations/index.js

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import API from 'foremanReact/API';
+
+import {
+  JOB_INVOCATIONS_GET_JOB_INVOCATIONS,
+  JOB_INVOCATIONS_POLLING_STARTED,
+  JOB_INVOCATIONS_JOB_FINISHED,
+} from '../../consts';
+
+const defaultJobInvocationsPollingInterval = 10000;
```

Comment:
Imo this should be 1000, otherwise it feels like a regression to what we had before